### PR TITLE
dmenu: add Chinese translation

### PR DESCRIPTION
**Pull request by sandylaw**
Description: add dmenu pages in pages.zh/common.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).
@ChungZH @KsRyY @whriedplanck

### CODE REVIEW

**Review by marchersimon — APPROVED** [review]

File: pages.zh/linux/dmenu.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# dmenu
+
+> 动态菜单。
+> 根据文本输入创建菜单，其中每一项都在新行中。
+
+- 显示`ls`命令输出的菜单：
```
Comment (bl-ue) [inline pages.zh/linux/dmenu.md:6]:
```suggestion
- 显示 `ls` 命令输出的菜单：
```

File: pages.zh/linux/dmenu.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- 显示包含自定义项目的菜单，并用新行（`\n`）分隔：
+
+`echo -e "{{red}}\n{{green}}\n{{blue}}" | dmenu`
+
+- 让用户在多个项目之间进行选择，然后将所选项目保存到文件中：
+
+`echo -e "{{red}}\n{{green}}\n{{blue}}" | dmenu > {{color.txt}}`
+
+- 在特定的监视器上启动`dmenu`：
```
Comment (bl-ue) [inline pages.zh/linux/dmenu.md:18]:
```suggestion
- 在特定的监视器上启动 `dmenu`：
```

File: pages.zh/linux/dmenu.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`ls | dmenu -m {{1}}`
+
+- 在屏幕底部显示`dmenu`：
```
Comment (bl-ue) [inline pages.zh/linux/dmenu.md:22]:
```suggestion
- 在屏幕底部显示 `dmenu`：
```

**Review by bl-ue — COMMENTED** [review]
Thank you for the translaton @sandylaw! I've left 3 comments below for you to review.  Per #5240 we put spaces around English in Chinese text.

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**